Source/Internal/API/APIEndpoints.swift

```diff
@@ -73,17 +76,31 @@ internal enum APIEndpoints {
         return apiKey
     }
 
+    // MARK: - Helper methods
+
+    /// Builds query parameters for the API endpoint `productDataCheck`
+    ///
+    /// - Parameter externalId: A string to represent the id that will be used to reference
+    ///  this product in Virtusize API
+    /// - Returns: An array of query items for the `URLComponents`
     private func dataCheckQueryItems(externalId: String) -> [URLQueryItem] {
         var queryItem: [URLQueryItem] = []
         queryItem.append(URLQueryItem(name: "apiKey", value: apiKey))
         queryItem.append(URLQueryItem(name: "externalId", value: externalId))
         queryItem.append(URLQueryItem(name: "version", value: "1"))
-        if let userID = Virtusize.userID {
-            queryItem.append(URLQueryItem(name: "externalUserId", value: userID))
+        if let externalUserID = Virtusize.userID {
+            queryItem.append(URLQueryItem(name: "externalUserId", value: externalUserID))
         }
         return queryItem
     }
 
+    /// Builds query parameters for the API endpoint `productMetaDataHints`
+    ///
+    /// - Parameters:
+    ///   - externalId: A string to represent the id that will be used to reference this product in Virtusize API
+    ///   - imageUrl: A string that is the image URL of the product, in order to populate the comparison view
+    ///   - storeId: An interger that represents the store id from the product data
```

Comment:
`integer`

Reply:
Oops, thanks. I just turned on the spell checking on Xcode. It should be more convenient to check typos beforehand.